packages/blade/src/components/Amount/amountTokens.ts

```diff
@@ -98,5 +320,6 @@ export {
   affixFontSizes,
   currencyPrefixMapping,
   currencyLocaleMapping,
-  currencyAbbreviationsMapping,
+  getCurrencyAbbreviations,
+  Currency,
 };
```

Comment:
Export type independently. We've had issues in the past in certain cases when types weren't exported independently
```suggestion
};

export type { Currency };
```

Reply:
We should add a lint rule for this.